proposals/0309-unlock-existential-types-for-all-protocols.md

```diff
@@ -146,7 +146,7 @@ func test(_ collection: RandomAccessCollection) {
 ```
 ___
 
-This way, a protocol or protocol extension member (method/property/subscript/initializer) may be used on an existential value iff the following criteria hold:
+This way, a protocol or protocol extension member (method/property/subscript/initializer) may be used on an existential value if and only if the following criteria hold:
 * The type of the invoked member (accessor — for storage declarations), as viewed in context of the *base type*, must **not** contain references to `Self` or `Self`-rooted associated types in [non-covariant](https://en.wikipedia.org/wiki/Covariance_and_contravariance_(computer_science)) position.
```

Comment:
While examining this section, I found the sentence structure here a bit hard to parse. I suggest we reverse the wording for easier understanding, which then also mirrors the outlined limitations [above](https://github.com/apple/swift-evolution/pull/1333/files#diff-431eac3f975001c52a9fc14a4300e856ad04ed161caa9c673e5c09ec989eeedfR23) 

```suggestion
This way, a protocol or protocol extension member (method/property/subscript/initializer) may be used on an existential value *unless*:
* The type of the invoked member (accessor — for storage declarations), as viewed in context of the *base type*, contains references to `Self` or `Self`-rooted associated types in [non-covariant](https://en.wikipedia.org/wiki/Covariance_and_contravariance_(computer_science)) position.
```

Feel free to discard this suggestion if you think it's not necessary.

Reply:
Your suggestion looks fine to me!

Reply:
Awesome, added it to the changes :)